docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3'
+services:
+  server:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile.server
+    volumes:
+      - .:/app/
+    ports:
+      - "2222:22"
+    networks:
+      network:
+        ipv4_address: 10.10.10.10
+    restart: always
```

Comment:
Can you add `platform: linux/amd64` key?

Reply:
Sure ;)